web/packages/teleport/src/Main/Main.tsx

```diff
@@ -188,7 +188,7 @@ export function Main(props: MainProps) {
   const banners: BannerType[] = alerts.map(alert => ({
```

Comment:
```suggestion
  const banners: BannerType[] = alerts.map((alert): BannerType => ({
```

You should also add a return type for the callback function, `BannerType`  in this case.  That would have helped typescript to catch this issue earlier and prevent similar one in the future.